data/recipes.js

```diff
@@ -466,7 +464,10 @@ const recipeData = [
     ],
     "name": "Elvis Pancakes",
     "tags": [
-      "side dish"
+      "side dish",
+      "breakfast",
+      "brunch",
+      "vegetarian"
```

Comment:
Nice addition here to add a few tags for search to pick up!! We'll need to beef up our search just a little bit to have it look inside tags, too, but we can make that happen pretty easily. Well done.